src/mina-signer/src/signature.ts

```diff
@@ -331,8 +331,9 @@ function getNetworkIdHashInput(network: NetworkId): [bigint, number] {
   switch (s) {
     case 'mainnet':
       return [networkIdMainnet, 8];
+    case 'devnet':
     case 'testnet':
```

Comment:
Are you thinking a log statement or a comment? It's a bit noisy during tests with the log

Reply:
I was thinking a log... why is it noisy?  Because we have tests still using `testnet`?  I suppose we can do a very soft deprecation for now and continue to fully support `testnet`, but eventually we will want to remove it from our tests.

Reply:
no logs from library code please!